features/main/always_identifier.feature

```diff
@@ -0,0 +1,117 @@
+Feature: AlwaysIdentifier support
+  In order to use a hypermedia API
+  As a client software developer
+  I need to be able to force identifier display of related entities
+
+  @createSchema
+  Scenario: Create a set of dummy with alwaysIdentifier attributes and check the jsonld formatted response
```

Comment:
Create a set of Dummy with the alwaysIdentifier attributes and check the JSON-LD formatted response